default/python/universe_generation/starsystems.py

```diff
@@ -1,10 +1,12 @@
-import sys
 import random
+import sys
+from itertools import product
+
 import freeorion as fo
+
 import planets
-import util
 import universe_tables
-from itertools import product
+import util
 
```

Comment:
...

Reply:
If you comment  on line that will not be changed you can leave checkbox:
```
- [ ] ....
```
This will allow to mark it as ready.
- [ ] ....